docs/configuration.md

```diff
@@ -1664,6 +1664,14 @@ Apart from these, the following properties are also available, and may be useful
         </td>
     </tr>
     <tr>
+        <td><code>spark.ssl.<particular protocol>.port</code></td>
```

Comment:
Agree; see https://github.com/apache/spark/pull/15652#discussion_r85307232  This should just refer to `spark.ssl.port` because the rest is documented above.